index.js

```diff
@@ -326,42 +326,50 @@ function addConfigHelpers (config) {
 }
 
 function approveDomains (config, cloud) {
+  var domainReg
+  if (config.hostname) {
+    // Dots in domains are normal but dots in a regexp would be replaced with "any character"
+    var regHost = (config.hostName || '').replace(/\./g, '\\.')
+    if (config.sites === 'per-archive') {
+      domainReg = new RegExp(`^((.+)-([^.]+)\\.)?${regHost}$`, 'g')
+    } else if (config.sites === 'per-user') {
+      domainReg = new RegExp(`^(()([^.]+)\\.)?${regHost}$`, 'g')
+    } else {
+      domainReg = new RegExp(`^${regHost}$`, 'g')
+    }
+  } else {
+    // Allow any domain
+    domainReg = /.?/g
```

Comment:
Should we throw in this case?

Reply:
Seems like this would just be a misconfiguration. We could probably throw an error as config validation around https://github.com/beakerbrowser/hashbase/blob/master/lib/config.js#L30

Reply:
It seems like the default configuration of most tests omits the hostname: https://travis-ci.org/beakerbrowser/hashbase/builds/327215556#L1654